aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -245,13 +306,12 @@ spec aptos_framework::staking_contract {
         voter: address,
         contract_creation_seed: vector<u8>,
     ): (signer, SignerCapability, OwnerCapability) {
-        // TODO: this function is normal in the local machine
-        // However, timeout in the github test
-
-        pragma verify = false;
-
+        // preconditions
         include stake::ResourceRequirement;
+
         let staker_address = signer::address_of(staker);
+        // verify account::create_resource_account()
```

Comment:
Could you rephrase the comment "// verify XYZ" with "// postconditions of XYZ"? (e.g., `// postconditions of account::create_resource_account()`) And, do the same for the other places?

Reply:
settled